src/dbt/powerschool/models/intermediate/int_powerschool__assignment_score_rollup.sql

```diff
@@ -32,6 +32,7 @@ with
         left join
             {{ ref("stg_powerschool__assignmentscore") }} as s
             on a.assignmentsectionid = s.assignmentsectionid
+            and {{ union_dataset_join_clause(left_alias="a", right_alias="s") }}
```

Comment:
remove: only needed for kipptaf models

Reply:
it's not joining across regions here. this is the model that gets created at the regional level

Reply:
sigh. so this is not why the average grade is not working? uuuuughhhhh